monai/transforms/utility/array.py

```diff
@@ -239,13 +241,13 @@ def __call__(self, img: torch.Tensor, meta_dict: Optional[Mapping] = None) -> to
         channel_dim = meta_dict.get("original_channel_dim")  # type: ignore
```

Comment:
```suggestion
        channel_dim = meta_dict.get("original_channel_dim", None)  # type: ignore
```
If `self.input_channel_dim` is not `None` but `img` was not a `MetaTensor`, it'll get converted into one with an empty meta dict, or if `img` is a `MetaTensor` without this value.

Reply:
updated -- btw if it's a regular dictionary the get method by default return None for the missing key https://docs.python.org/3/library/stdtypes.html?highlight=get#dict.get